lessonbuilder/api/src/java/org/sakaiproject/lessonbuildertool/ChecklistItemStatus.java

```diff
@@ -0,0 +1,39 @@
+/**********************************************************************************
+ * $URL: $
+ * $Id: $
+ ***********************************************************************************
+ *
+ * Author: David P. Bauer, dbauer1@udayton.edu
+ *
+ * Copyright (c) 2016 University of Dayton
+ *
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ **********************************************************************************/
```

Comment:
This is not the ECLv2 header that is distributed with Sakai.

Can you please run the maven license goal to update these headers.